### PR TITLE
STORM-1365:  Small improvements in Travis scripts

### PR DESCRIPTION
**Pull request by ashnazg**

### CODE REVIEW

File: dev-tools/travis/travis-script.sh [hunk 1]
```diff
@@ -31,7 +25,6 @@ cd ${STORM_SRC_ROOT_DIR}
 # We should be concerned that Travis CI could be very slow because it uses VM
 export STORM_TEST_TIMEOUT_MS=150000
 
-# We now lean on Travis CI's implicit behavior, ```mvn clean install -DskipTests``` before running script
```
Comment (d2r) [inline dev-tools/travis/travis-script.sh]: Was this comment incorrect or unhelpful?

**Comment by d2r** [comment]
Seems OK to me.  I think the test failure was unrelated.

File: dev-tools/travis/travis-script.sh (hunk 1 above)
Comment (revans2) [inline dev-tools/travis/travis-script.sh]: It is incorrect, because we don't do that any more.

**Comment by revans2** [comment]
+1 looks good to me. @ashnazg you might want to rebase on master I think @d2r pulled out something that went in that was causing test failures, so you might get a clean run in travis now.

**Comment by d2r** [comment]
+1 pending CI

File: dev-tools/travis/travis-script.sh (hunk 1 above)
Comment (d2r) [inline dev-tools/travis/travis-script.sh]: Sounds good.

**Comment by jerrypeng** [comment]
+1

**Comment by harshach** [comment]
+1

**Comment by harshach** [comment]
Thanks @ashnazg merged into master.

**Pull request merged**